data.py

```diff
@@ -0,0 +1,168 @@
+from datetime import datetime
+import requests
+import schema
+from bs4 import BeautifulSoup
+
+URL = 'https://now.dining.cornell.edu/api/1.0/dining/eateries.json'
+PAYMENT_METHODS = {
+  'swipes': 'Meal Plan - Swipe',
+  'brbs': 'Meal Plan - Debit',
+  'cornell_card': 'Cornell Card',
+  'credit': 'Major Credit Cards',
+  'mobile': 'Mobile Payments'
+}
+
+eateries = {}
+operating_hours = {}
+events = {}
+menus = {}
+items = {}
+
+def start_update():
+  try:
+    print('[{0}] Updating data'.format(datetime.now()))
+    result = requests.get(URL)
+    data_json = result.json()
+    parse(data_json)
+    data = schema.Data()
+    data.update_data(
+      eateries=eateries,
+      operating_hours=operating_hours,
+      events=events,
+      menus=menus,
+      items=items)
+  except Exception as e:
+    print 'data update failed:', e
+
+
+def parse(data_json):
+  global eateries
+  for eatery in data_json['data']['eateries']:
+    new_eatery = schema.EateryType(
+      id=eatery['id'],
+      slug=eatery['slug'],
+      name=eatery['name'],
+      name_short=eatery['nameshort'],
+      about=eatery['about'],
+      about_short=eatery['aboutshort'],
+      image_url=parse_image_url(eatery['about']),
+      payment_methods=parse_payment_methods(eatery['payMethods']),
+      # calender_id=eatery['googleCalenderId'],
+      location=eatery['location'],
+      operating_hours=parse_operating_hours(eatery['operatingHours'], eatery['id']),
+      coordinates=parse_coordinates(eatery['coordinates']),
+      campus_area=parse_campus_area(eatery['campusArea'])
+      )
+    eateries[new_eatery.id] = new_eatery
+
+
+def parse_image_url(string):
+  soup = BeautifulSoup(string, 'lxml')
+  found = soup.find('img')
+  if found is None:
+    return 'http://parse_image_url_failed'
+  image_url = found['src']
+  return image_url
+
+def parse_payment_methods(payment_methods):
+  new_payment_methods = schema.PaymentMethodsType()
+
+  if any(method['descrshort'] == PAYMENT_METHODS['swipes'] for method in payment_methods):
+    new_payment_methods.swipes = True
+  else:
+    new_payment_methods.swipes = False
+  if any(method['descrshort'] == PAYMENT_METHODS['brbs'] for method in payment_methods):
+    new_payment_methods.brbs = True
+  else:
+    new_payment_methods.brbs = False
+  if any(method['descrshort'] == PAYMENT_METHODS['credit'] for method in payment_methods):
+    new_payment_methods.credit = True
+    new_payment_methods.cash = True
+  else:
+    new_payment_methods.credit = False
+    new_payment_methods.cash = False
+  if any(method['descrshort'] == PAYMENT_METHODS['cornell_card'] for method in payment_methods):
+    new_payment_methods.cornell_card = True
+  else:
+    new_payment_methods.cornell_card = False
+  if any(method['descrshort'] == PAYMENT_METHODS['mobile'] for method in payment_methods):
+    new_payment_methods.mobile = True
+  else:
+    new_payment_methods.mobile = False
```

Comment:
can just set the variable equal to the value `any(...)` since that's already the boolean you want